plugins/audit-log-node/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Can you update the [team/rhdh renovate rule](https://github.com/janus-idp/backstage-plugins/blob/main/.github/renovate.json#L191) to add this new plugin so it gets scanned?  It'll save me the trouble of opening a new PR :pray:

e.g.

```

      "matchFileNames": [
        "plugins/audit-log-node*/**",   
        ...
      ],
      "additionalBranchPrefix": "rhdh ",
      "addLabels": [
        "team/rhdh"
      ]
```

Reply:
Will do